2019-08-07-all-hands-meeting.md

```diff
@@ -0,0 +1,117 @@
+# 2019-08-07 Hypha Worker Co-op: All Hands Meeting
+
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
+
+Time:      Wed, August 7 @ 5:00-7:00pm  
+Location:  https://meet.jit.si/hyphacoop  
+Attending: pc, dc, ei, bl, yj (briefly)  
+Notetaker: :raising_hand: ei, yj, gi, pc, dc, uv, bl  
+
+## Agenda
+
+| Item                                            | Time (min) |
+|:------------------------------------------------|-----------:|
+| [Previous Action Items][tasks]                  |         35 |
+| [Announcements](#Announcements)                 |          5 |
+| [Working Group Updates](#Working-Group-Updates) |         15 |
+| [Project Updates](#Project-Updates)             |         20 |
+| [Discussions](#Discussions)                     |         45 |
+
+## Notes
+
+- dc ideologically opposed to naked headers
+- people aren't sure yet about this new meeting template
+    - seems light on notes
+    - dc: should we go through all tasks? seems much
+    - pc: prioritizing label? +1 dc
+    - patcon: feel like things were accomplished. but won't have notes
+    - dc: kinda opposite, unsure what has been accomplished
+
+### Announcements
+
+- Saturday noon until whenever BBQ at Ben's (add to calendar + ping uv), pc + el + dc
+    - this is a post-burnout therapy session, no talking biz
+
+### Working Group Updates
+
+- [Business Planning][biz-wg]
+    - dc: we really need to meet, operating cost needs to land
+    - dc: gi wants to do branding in august
```

Comment:
https://github.com/hyphacoop/organizing/issues/77

Testing issue crosslinking

Reply:
Do you mean for these to be added to notes? What is the purpose of these comments?

Reply:
The issue will have crosslinks to what meetings its discussed in

Reply:
Could immediately resolve as part of process. Or we could figure out another way to crosslink (dated labels were discussed, but they feel heavy in github)

Reply:
Ahhh I think this works

Reply:
Ahhh I think this works

Reply:
Ahhh I think this works

Reply:
Ahhh I think this works